languages/elixir/exercises/concept/recursion/.docs/instructions.md

```diff
@@ -0,0 +1,55 @@
+You're an avid bird watcher that keeps track of how many birds have visited your garden on any given day.
+
+You decided to bring your bird watching to a new level and implement a few tools that will help you track and process the data.
+
+You have chosen to store the data as a list of integers. The first number in the list is the number of birds that visited your garden today, the second yesterday, and so on.
+
+## 1. Check how many birds visited today
```

Comment:
I dropped the first step of the C# exercise because it's about returning hardcoded data. I find that a bit weird, but also it doesn't teach anything about recursion.

https://github.com/exercism/v3/blob/ded2bcfb25875a033db23110b15f771bd1877368/languages/csharp/exercises/concept/arrays/.docs/instructions.md#L5-L12

Reply:
FYI the hard-coded data task was there to force students to define an array from scratch, not work with an existing array. We found that to be important as we really want students to be familiar with the syntax of defining arrays. 

I totally agree that it removing it from this exercise makes perfect sense. And this is exactly how ports should work: the original should adapted to best fit the track. 👍